docs/src/pages/[platform]/connected-components/authenticator/customization/override-function-calls/example.angular.mdx

```diff
@@ -0,0 +1,107 @@
+export let prevVersion = PREV_MAJOR_VERSIONS['angular'];
+
+_app.component.html_
+
+```html{1}
+<amplify-authenticator
+  [services]="services"
+  initialState="signUp"
+>
+  <ng-template
+    amplifySlot="authenticated"
+    let-user="user"
+    let-signOut="signOut"
+  >
+    <h2>Welcome, {{ user.username }}!</h2>
+    <button (click)="signOut()">Sign Out</button>
+  </ng-template>
+</amplify-authenticator>
+
+```
+
+_app.component.ts_
+<Tabs.Container defaultValue={latestVersion}>
+  <Tabs.List>
+    <Tabs.Item value={latestVersion}>ui-angular v{latestVersion} (latest)</Tabs.Item>
+    <Tabs.Item value={prevVersion}>ui-angular v{prevVersion}</Tabs.Item>
+  </Tabs.List>
+  <Tabs.Panel value={latestVersion}>
+    In `@aws-amplify/ui-angular` version {latestVersion}, `Auth` function calls are imported directly as shown below. 
+    ```js{3}
+    import { Component, OnInit } from '@angular/core';
+    import { Amplify } from 'aws-amplify';
+    import { signUp } from 'aws-amplify/auth';
+    import awsExports from './aws-exports';
+
+    @Component({
+      selector: 'sign-up-with-email',
+      templateUrl: 'sign-up-with-email.component.html',
+    })
+    export class SignUpWithEmailComponent implements OnInit {
+      constructor() {
+        Amplify.configure(awsExports);
+      }
+      services = {
+        async handleSignUp(formData: Record<string, any>) {
+          let { username, password, options } = formData;
+          // custom username and email
+          username = username.toLowerCase();
+          const customEmail = options?.userAttributes?.email?.toLowerCase();
+          return signUp({
+            username,
+            password,
+            options: {
+              ...formData?.options,
+              userAttributes: {
+                ...formData?.options?.userAttributes,
+                email: customEmail,
+              },
+            },
+          });
+        },
+      };
+    }
+    ```
+  </Tabs.Panel>
+  <Tabs.Panel value={prevVersion}>
+    <Alert role="none" variation="warning" heading="Sign Up Auto Sign In">
+    If you're overriding `signUp` in `@aws-amplify/ui-angular` **version {prevVersion} or earlier**, you must include the `autoSignIn` key and set `enabled` to true, as shown in the example below.
+    </Alert>
+    In `@aws-amplify/ui-angular` version {prevVersion} or earlier, `Auth` function calls are imported from the `Auth` object.
+    ```jsx{2,25}
+      import { Component, OnInit } from '@angular/core';
+      import { Amplify, Auth } from 'aws-amplify';
+      import awsExports from './aws-exports';
+
+      @Component({
+        selector: 'sign-up-with-email',
+        templateUrl: 'sign-up-with-email.component.html',
+      })
+      export class SignUpWithEmailComponent implements OnInit {
+        constructor() {
+          Amplify.configure(awsExports);
+        }
+
+        services = {
+        async handleSignUp(formData: Record<string, any>) {
```

Comment:
Indentation looks off here. Also prefer `input` over `formData` (same feedback for any other usages of `formData` that are used in `handle*` signatures)

Reply:
Fixed indentation. And even for the previous version examples? I assumed that was something that changed in the later version but was it always `input`?